shell/common/animator.cc

```diff
@@ -27,22 +27,27 @@ Animator::Animator(Delegate& delegate,
       waiter_(std::move(waiter)),
       last_begin_frame_time_(),
       dart_frame_deadline_(0),
+#if FLUTTER_SHELL_ENABLE_METAL
+      layer_tree_pipeline_(fml::MakeRefCounted<LayerTreePipeline>(2)),
+#else   // FLUTTER_SHELL_ENABLE_METAL
       // TODO(dnfield): We should remove this logic and set the pipeline depth
-      // back to 2 in this case. See https://github.com/flutter/engine/pull/9132
-      // for discussion.
+      // back to 2 in this case. See
+      // https://github.com/flutter/engine/pull/9132 for discussion.
       layer_tree_pipeline_(fml::MakeRefCounted<LayerTreePipeline>(
           task_runners.GetPlatformTaskRunner() ==
                   task_runners.GetGPUTaskRunner()
               ? 1
               : 2)),
+#endif  // FLUTTER_SHELL_ENABLE_METAL
       pending_frame_semaphore_(1),
       frame_number_(1),
       paused_(false),
       regenerate_layer_tree_(false),
       frame_scheduled_(false),
       notify_idle_task_id_(0),
       dimension_change_pending_(false),
-      weak_factory_(this) {}
+      weak_factory_(this) {
```

Comment:
Nit: This may require a `clang-format` pass.

Reply:
This is how clang-format is putting it - maybe it's confused by the preprocessor macros?